File: java_project/NeonNumber.java
public class NeonNumber {
    public static void main(String[] args) {
        
        int number = 9;

        
        int square = number * number;

        System.out.println(square);

        
        int sumOfDigits = 0;
        int tempSquare = square; 

        while (tempSquare > 0) {
            int digit = tempSquare % 10;
            sumOfDigits += digit; 
            tempSquare /= 10; 
        }

        
        if (sumOfDigits == number) {
            System.out.println(number + " is a Neon Number.");
        } else {
            System.out.println(number + " is NOT a Neon Number.");
        }
    }
}

File: python_project/NeonNumber.py
""" n = 9
sum = 0
square =  n*n
temp = square
print (square) 
while temp>0:
    digit = temp%10
    sum = sum + digit
    temp = temp/10
   

print (sum)
print (digit)
if sum == n:
    print (n,  " is a Neon Number")
else:
    print (n, " is not a Neon Number") """

n = 9
sum = 0
square = n * n
temp = square

print("Square:", square) 

while temp > 0:
    digit = temp % 10      # Pull the last digit from temp, not square
    sum = sum + digit      # Add it to the running sum
    temp = temp // 10      # Use // for integer floor division

print("Sum of digits:", sum)
if sum == n:
    print(n, "is a Neon Number")
else:
    print(n, "is not a Neon Number")